v19.2/configure-zone.md

```diff
@@ -40,7 +40,12 @@ Adding replication zones for rows and secondary indexes is an [enterprise-only](
 
 ## Required privileges
 
-Currently, only the `root` user can configure replication zones.
+If the target is a [`system` range](#create-a-replication-zone-for-a-system-range), the [`system` database](show-databases.html#preloaded-databases), or a table in the `system` database, the user must be an [`admin`](authorization.html#create-and-manage-roles). For all other databases and tables, the user must have the [CREATE](grant.html#supported-privileges) privilege on the target database or table.
+
+{{site.data.alerts.callout_info}}
+Required privileges for `CONFIGURE ZONE` statements in CockroachDB v19.2 may be backward-incompatible for users running scripted statements with restricted permissions in v19.1 and earlier.<br/>To add the necessary permissions, use [`GRANT` &lt;privileges&gt;](../v19.2/grant.html) or [`GRANT` &lt;roles&gt;](../v19.2/grant-roles.html) as a user with an admin role. For example, to grant a user the admin role, run `GRANT admin TO <user>`. To grant the `CREATE` privilege on a database or table, run `GRANT CREATE ON [DATABASE | TABLE] <name> TO <user>`.
```

Comment:
~I think this should be a red box since this is potentially a breaking change

Reply:
It's a tad awkward to have all this inside a blue box, but I agree with the choice. Suggest adding line breaks before "For example" and again before "To grant the CREATE privilege" to make it easier to scan (since those lines are each separate tasks for the user).